pkgs/applications/misc/tandoor-recipes/default.nix

```diff
@@ -150,6 +157,12 @@ python.pkgs.pythonPackages.buildPythonPackage rec {
 
     tests = {
       inherit (nixosTests) tandoor-recipes;
+      runTandoorRecipes = runCommand "run-${pname}" {} ''
+        ${tandoor-recipes}/bin/tandoor-recipes > $out
+      '';
+      runManagePy = runCommand "run-${pname}" {} ''
+        ${tandoor-recipes}/lib/tandoor-recipes/manage.py > $out
+      '';
```

Comment:
The `tandoor-recipes` test already checks that the manage script works, as part of bringing up the module.

Those tests are redundant with that one.